### PR TITLE
 finished ticket 11 more features

### PR DESCRIPTION
**Pull request by Gary-Huynh**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -169,6 +170,39 @@ describe("GET /api/articles?sort_by=votes / article_id/created_at/comment_count"
...
+            expect(body.articles).toBeSortedBy("created_at",{ascending:true})
+        })
+    })
+    test("400: /api/articles?order=banana should return a 400 bad request as it is not a valid order",()=>{
+        return request(app).get("/api/articles?order=banana").expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe("Bad Request invalid order")
+        })
+    })
+})
```
Comment (NRMSMITH) [inline __tests__/app.test.js:211]: Three more scenarios to envision:
-What if I sort_by a valid column: ?sort_by=votes
-What if I sort_by an invalid column: ?sort_by=bananas
-What if I query a topic that exists but has not articles: ?topic=paper

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
A few more scenarios to run tests for !! Please resubmit PR once done :)

File: __tests__/app.test.js (hunk 1 above)
Comment (Gary-Huynh) [reply __tests__/app.test.js:211]: the first two are already tested for in lines 158 - line 170

Comment (NRMSMITH) [reply __tests__/app.test.js:211]: Ah! For some reason they're not actually a part of this PR - so they were in hidden view! Strange 👻 It's all good here anyway!

**Review by NRMSMITH — APPROVED** [review]
💯 💯 Amazing, lovely, well done!

**Pull request merged**